internal/service/amp/scraper.go

```diff
@@ -47,13 +47,13 @@ func newScraperResource(_ context.Context) (resource.ResourceWithConfigure, erro
 
 	r.SetDefaultCreateTimeout(30 * time.Minute)
 	r.SetDefaultDeleteTimeout(20 * time.Minute)
+	r.SetDefaultUpdateTimeout(2 * time.Minute)
```

Comment:
Registry documentation also needs updated to include this new value.

Reply:
Done.